### PR TITLE
Armor plate component improvements

### PR DESCRIPTION
**Pull request by ChungusGamer666**
## About The Pull Request

Just small improvements to the component, fixing it up for current tg standards.

## Why It's Good For The Game
Better code.

### CODE REVIEW

File: code/datums/components/armor_plate.dm [hunk 1]
```diff
@@ -1,91 +1,105 @@
 /datum/component/armor_plate
+	/// Current amount of upgrades
 	var/amount = 0
+	/// Maximum amount of upgrades
 	var/maxamount = 3
```
Comment (ArcaneMusic) [inline code/datums/components/armor_plate.dm]: You have mixed use of `maxamount` and `max_amount`. `max_amount` would be preferred variable-name wise.

Comment (ChungusGamer666) [reply code/datums/components/armor_plate.dm]: I done goofed, pushing fixes.

**Comment by ChungusGamer666** [comment]
please add code improvement label :)

**Comment by ChungusGamer666** [comment]
i give up

**Pull request closed without merging**